config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -67,6 +67,38 @@ presubmits:
               cpu: 4
               memory: "36Gi"
 periodics:
+  - interval: 1h
+    name: ci-kubernetes-unit
+    annotations:
+      fork-per-release: "true"
+      testgrid-num-failures-to-alert: '3'
+      testgrid-alert-stale-results-hours: '24'
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: sig-release-master-informing
```

Comment:
```suggestion
      testgrid-dashboards: sig-release-master-blocking
      testgrid-tab-name: unit-test-master
```

Reply:
ref: https://github.com/kubernetes/sig-release/blob/master/release-blocking-jobs.md#promoting-or-demoting-jobs

/sig release
/assign @hasheddan @saschagrunert @thejoycekung @palnabarun 
I'd like to defer to SIG Release TLs, v1.21 CI Signal and Lead here ... but here's my pitch to start at blocking.

IMO this is ~equivalent to the `bazel-test` tab that _was_ on release-blocking (eg: https://testgrid.k8s.io/sig-release-1.20-blocking#bazel-test-1.20).  Given that we already have a merge-blocking variant of this job, we're already likely going to see @kubernetes/ci-signal keeping tabs on unit test flakes (community heroics aside)

So while ordinarily I'd advocate for N days/weeks in informing before promoting prior to test freeze, I feel like we don't save much effort by doing so.

Reply:
... that said a quick trial in informing followed by promote is probably the more reasonable thing I'm asking for vs. just slam into blocking

Reply:
@spiffxp I would be in favor of a short time on `informing` before promotion to `blocking` :+1: Per the [promotion guidelines](https://github.com/kubernetes/sig-release/blob/master/release-blocking-jobs.md#promoting-or-demoting-jobs), we'll need @palnabarun to officially sign off on this.

Reply:
I lean towards following the procedure so as not to open a path to exceptions in the future. I think we can afford to add this to release blocking after a short time given we only _just_ entered code freeze and have blocking presubmit coverage (which did go through a promotion path as well, even if theoretically equivilant)

Reply:
Keeping this in `informing` for a short period of time makes sense. Let's promote this to `blocking` after soaking in for a few days.